src/ring_theory/coprime.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2020 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau, Ken Lee, Chris Hughes
+-/
+
+import algebra.big_operators
+import data.fintype.basic
+import data.set.disjointed
+
+/-!
+# Coprime elements of a ring
+
+## Main definitions
+
+* `is_coprime x y`: that `x` and `y` are coprime, defined to be the existence of `a` and `b` such
+that `a * x + b * y = 1`.
+
+-/
+
+open_locale classical big_operators
+
+universes u v
+
+variables {R : Type u} [comm_semiring R] (x y z : R)
+
+/-- The proposition that `x` and `y` are coprime, defined to be the existence of `a` and `b` such
+that `a * x + b * y = 1`. -/
```

Comment:
Please mention in the docstring that with this definition elements with no common divisors are not necessarily coprime, e.g., the multivariate polynomials `x₁` and `x₂` are not coprime.

Reply:
done